Client/Cliqz/Frontend/Browser/FreshtabViewController.swift

```diff
@@ -220,6 +220,7 @@ class FreshtabViewController: UIViewController, UIGestureRecognizerDelegate {
 
 	private func loadTopsites() {
 		self.loadTopSitesWithLimit(15)
+        self.topSitesCollection.reloadData()
```

Comment:
Why topSites should be rendered before loading the data? It should be after the reloading the data